templates/beego/files/conf/conf.go

```diff
@@ -0,0 +1,16 @@
+package conf
+
+import (
+  psh "github.com/platformsh/config-reader-go/v2"
+)
+
+var Config = createConfigObj()
+
+func createConfigObj() *psh.RuntimeConfig {
+  // The Config Reader library provides Platform.sh environment information mapped to Go structs.
+  config, err := psh.NewRuntimeConfig()
+  if err != nil {
+  	panic("Not in a Platform.sh Environment.")
```

Comment:
`panic(err)`.  I discovered the hard way that the err may not actually be for not being in a Psh environment, and we shouldn't suppress the actual error message.